docs/modules/ROOT/pages/erc20.adoc

```diff
@@ -241,3 +241,13 @@ mod MyToken {
 This contract expects a `decimals` argument in the constructor and uses an internal function to write the decimals to storage.
 Note that the `decimals` state variable must be defined in the contract's storage because this variable does not exist in the component offered by OpenZeppelin Contracts for Cairo.
 It's important to include a custom ERC20 metadata implementation and NOT use the Contracts for Cairo `ERC20MetadataImpl` in this specific case since the `decimals` method will always return `18`.
+
+== Preset ERC20 contract
```

Comment:
https://github.com/OpenZeppelin/cairo-contracts/pull/803#issuecomment-1798461724

As mentioned in the comment, I think we should keep this in the references side of the documentation, and not in the understanding section.